code/modules/power/cell.dm

```diff
@@ -103,9 +103,9 @@
 	return (FIRELOSS)
 
 /obj/item/stock_parts/cell/on_reagent_change(changetype)
-	rigged = !isnull(reagents.has_reagent(/datum/reagent/toxin/plasma, 5)) //has_reagent returns the reagent datum
-	..()
+	rigged = reagents.has_reagent(/datum/reagent/toxin/plasma, 5))

```

Comment:
should add a var to handle if rigged was set by corrupt() so we don't end up overriding that if the chems were removed